vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml

```diff
@@ -20,6 +20,14 @@
   set_fact:
     config_backup_folder_path: "{{ root_folder }}/data/arkcase-home/.arkcase-backup-{{ ansible_date_time.iso8601 }}"
 
+- name: Delete all but last 5 backups
+  become: yes
+  become_user: arkcase
+  shell: |
+         cd {{ root_folder }}/data/arkcase-home
+         ls -atp | grep ".arkcase-backup*"| tail -n +6 | xargs -d '\n' -r rm -rf
+  when: arkcase_config_path.stat.exists
+  
```

Comment:
Two other approaches will be more ansible-like and easier to read:

```yaml
shell: ls -atp ....
args:
  chdir: "{{ root_folder}}..."
```

or: 

```yaml
shell: ls -atp {{ root_folder }}/data/arkcase-home | grep ".arkcase-backup*" ...
```

Reply:
Done. Thanks.

Reply:
Done. Thanks.